<ui version="4.0" >
 <author>Stefan BÃ¶hmann</author>
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>538</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="KButtonGroup" name="kbuttongroup1" >
     <property name="title" >
      <string>Tea List</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeView" name="tealistTreeView" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="itemsExpandable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QToolButton" name="newButton" >
          <property name="text" >
           <string>N</string>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeButton" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>D</string>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="upButton" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>U</string>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="downButton" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>D</string>
          </property>
          <property name="icon" >
           <iconset/>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="teaPropertiesGroup" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Tea Properties</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4" >
       <widget class="KLineEdit" name="teaNameEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="minutesSpin" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximum" >
         <number>1440</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QSpinBox" name="secondsSpin" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="maximum" >
         <number>59</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>sec</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="KButtonGroup" name="kbuttongroup2" >
     <property name="title" >
      <string>Action</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="notificationCheckBox" >
          <property name="text" >
           <string>Notification</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="notificationButton" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Configure &amp;Notifications...</string>
          </property>
          <property name="default" >
           <bool>false</bool>
          </property>
          <property name="flat" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="popupCheckBox" >
        <property name="text" >
         <string>Popup</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="autohideCheckBox" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Auto hide popup after</string>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="autohideSpinBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>300</number>
          </property>
          <property name="value" >
           <number>30</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>seconds.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>151</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="reminderCheckBox" >
          <property name="text" >
           <string>Reminder every</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="reminderSpinBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>300</number>
          </property>
          <property name="value" >
           <number>60</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>seconds.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>181</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QCheckBox" name="visualizeCheckBox" >
     <property name="text" >
      <string>Visualize progress in icon tray</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>notificationCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>notificationButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>88</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel" >
     <x>314</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autohideCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>autohideSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>222</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reminderCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>reminderSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>108</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel" >
     <x>217</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
